vagrant/base.sh

```diff
@@ -38,8 +38,25 @@ if [ -z `which javac` ]; then
 fi
 
 chmod a+rw /opt
-if [ ! -e /opt/kafka ]; then
-    ln -s /vagrant /opt/kafka
+if [ -h /opt/kafka ]; then
+    # reset symlink
+    rm /opt/kafka
+fi
+ln -s /vagrant /opt/kafka
+
+version=0.8.2.1
```

Comment:
Might just want to change this into a function right now since you've already made it work with the parameter. It won't be long before we need to add more versions here.